tensorflow/lite/python/interpreter.py

```diff
@@ -192,6 +193,9 @@ def __init__(self,
       experimental_delegates: Experimental. Subject to change. List of
         [TfLiteDelegate](https://www.tensorflow.org/lite/performance/delegates)
         objects returned by lite.load_delegate().
+      num_threads: Set the number of threads used by TFLite kernels.
+        If not set, kernels are running single-threaded. Note that currently,
```

Comment:
This should instead say:

"If not set, the number of threads used for inference is implementation-dependent. Note that..."

Reply:
Thanks, done.